### PR TITLE
Remove data from getEvents

### PR DESCRIPTION
**Pull request by MironowDW**

### CODE REVIEW

File: src/Snowcap/Emarsys/Client.php [hunk 1]
```diff
@@ -553,9 +553,9 @@ public function getEmailCategories(array $data)
      * @param array $data
```
Comment (Moinax) [inline src/Snowcap/Emarsys/Client.php]: Are you sure it is not used ? Anyway, please remove phpdoc

**Comment by tim-bezhashvyly** [comment]
No, it is not.

https://dev.emarsys.com/suite/external_events/external_event_list.html

**Comment by Moinax** [comment]
Ok then, remove the phpdoc and I will be pleased to merge your PR :)

**Comment by MironowDW** [comment]
Oops

**Pull request merged**